packages/api/package.json

```diff
@@ -16,11 +16,17 @@
   "devDependencies": {
     "@types/mocha": "^8.2.2",
     "assert": "^2.0.0",
+    "buffer": "^6.0.3",
     "playwright-test": "^5.0.0",
-    "webpack": "^5.39.1",
+    "process": "^0.11.10",
+    "stream-browserify": "^3.0.0",
+    "webpack": "^5.42.0",
     "webpack-cli": "^4.7.2"
   },
   "dependencies": {
+    "@magic-ext/oauth": "^0.8.0",
+    "@magic-sdk/admin": "^1.3.0",
+    "@web3-storage/db": "^0.0.0",
```

Comment:
@alanshaw on the npms... do we want 
- `@web3-storage/db` 
- `@web3storage/db` - matches the hyphenless @nftstorage 
- `@dotstorage/web3.storage-db` - we talked about a `dotstrorage` org, but then there would be times you'd want to differentiate which app a module was actually for.

I guess we would not actually publish the db to npm, but we'd probably want to use a prefix that we controlled regardless, so this is mostly for illustration purposes

Reply:
am gonna go with `@web3-storage/db` as you have it.

Reply:
We had `nftstorage` on github so I made `@nftstorage` on npm too for consistency and so we have `web3-storage` on github here so I went for `@web3-storage` on npm. Even though we won't publish any modules...yet, we might want to publish web3.storage related packages to `@web3-storage` eventually so worth land grabbing. WDYT?

The other packages use a prefix e.g. `web3.storage-api` - we should probably change these also.

I think it is ok for the JS client to use `web3.storage` is it's name though.